source/_posts/2018-01-04-adam-0-dot-23-dot-0-released-plus-avocado-cannoli-and-deca-releases.markdown

```diff
@@ -0,0 +1,640 @@
+---
+layout: post
+title: "ADAM 0.23.0 Released (+ Avocado and DECA releases)"
+date: 2018-01-04 09:47:53 -0800
+comments: true
+categories: 
+---
+
+We are excited to announce the availability of the ADAM 0.23.0 release, along
+with releases of Avocado germline variant caller (release 0.1.0) and the DECA
+copy number variant caller (release 0.2.0). These releases contain an extensive
+number of feature additions, performance improvements, and bug patches, with
+over 375 issues and pull requests merged since the last ADAM release. Some of
```

Comment:
375 issues and pull requests merged &rarr; 375 issues closed and pull requests merged